music/demos/ddsp_tone_transfer.ts

```diff
@@ -0,0 +1,253 @@
+/**
+ * @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as mm from '../src/index';
+import { SPICE } from '../src/index';
+import { AudioFeatures, ModelValues } from '../src/ddsp/interfaces';
+
+enum MODEL {
+  VIOLIN = 'violin',
+  TENOR_SAXOPHONE = 'tenor_saxophone',
+  TRUMPET = 'trumpet',
+  FLUTE = 'flute',
+}
+
+export const PRESET_MODEL_URL =
+  'https://storage.googleapis.com/magentadata/js/checkpoints/ddsp';
+
+export const PRESET_MODELS = {
+  [MODEL.VIOLIN]: {
+    checkpointUrl: `${PRESET_MODEL_URL}/${MODEL.VIOLIN}`,
```

Comment:
If you have the URL, you shouldn't need these settings, because you can just read them from settings.json off of cloud in the model directory, right?